bin/coin-hive

```diff
@@ -31,7 +31,8 @@ if (argv.help || argv.h) {
     interval: argv.interval || process.env.COINHIVE_INTERVAL || defaults.interval ,
     port: argv.port || process.env.COINHIVE_PORT || defaults.port,
     host: argv.host || process.env.COINHIVE_HOST || defaults.host ,
-    threads: argv.threads || process.env.COINHIVE_THREADS || defaults.threads
+    threads: argv.threads || process.env.COINHIVE_THREADS || defaults.threads,
+    proxy: argv.proxy
```

Comment:
what about configuring this through env variables?

Reply:
+1